coverage/pom.xml

```diff
@@ -57,6 +57,11 @@
         <groupId>org.jacoco</groupId>
         <artifactId>jacoco-maven-plugin</artifactId>
         <version>0.8.8</version>
+        <configuration>
```

Comment:
Here u need to add workflow-examples as a dependency, no?

Reply:
You're right! I forgot to push the change: thanks!